src/promptflow/promptflow/_sdk/operations/_local_storage_operations.py

```diff
@@ -348,14 +353,14 @@ def load_detail(self) -> Dict[str, list]:
                 if line_run_record_file.suffix.lower() != ".jsonl":
                     continue
                 with open(line_run_record_file, mode="r", encoding=DEFAULT_ENCODING) as f:
-                    new_runs = [json.loads(line)["run_info"] for line in list(f)]
+                    new_runs = [json_loads(line)["run_info"] for line in list(f)]
```

Comment:
Seems this is now only avaliable for visualize, does `show-details` also need this improvement? And I also see the `load_detail` function get used in serving code, shall we also change the interface there?

Reply:
similar to my above comment, I expect this change only happen in visualize, where we will need to communicate with JS; in pure Python world (maybe also C#, not sure because seems PFS can handle this good), it works good, so don't need to do extra thing.